x/auth/keeper/grpc_query.go

```diff
@@ -83,3 +83,28 @@ func (ak AccountKeeper) Params(c context.Context, req *types.QueryParamsRequest)
 
 	return &types.QueryParamsResponse{Params: params}, nil
 }
+
+// ModuleAccounts returns all the existing Module Accounts
+func (ak AccountKeeper) ModuleAccounts(c context.Context, req *types.QueryModuleAccountsRequest) (*types.QueryModuleAccountsResponse, error) {
+	if req == nil {
+		return nil, status.Error(codes.InvalidArgument, "empty request")
+	}
+
+	ctx := sdk.UnwrapSDKContext(c)
+
+	var a []*codectypes.Any
```

Comment:
To fix the lint error:

```suggestion
	a := make([]*codectypes.Any, 0, len(ak.permAddrs))
```